resources/views/device-group/index.blade.php

```diff
@@ -57,6 +57,39 @@ class="fa fa-trash" aria-hidden="true"></i></button>
             </div>
         </div>
     </div>
+    <div class="container-fluid">
+        <div id="manage-device-groups-panel" class="panel panel-default">
+            <div class="panel-heading">
+                <h4 class="panel-title">
+                @lang('Ungrouped Devices') ({{ $ungrouped_devices->count() }})
+                </h4>
+            </div>
+            <div class="panel-body">
+                <div class="table-responsive">
+                    <table id="ungrouped-devices-table" class="table table-condensed table-hover">
+                        <thead>
+                        <tr>
+                            <th>@lang('Vendor')</th>
```

Comment:
Set the width of this column so it is as small as possible:  `<th style="width:32px">@lang('Vendor')</th>`

Reply:
changed